src/pages/my/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { GetServerSideProps } from 'next';
+import { Heading } from '@adobe/react-spectrum';
+import { FormattedMessage as Msg } from 'react-intl';
+
+import MyHomeLayout from '../../components/layout/MyHomeLayout';
+import { PageWithLayout } from '../../types';
+import { scaffold } from '../../utils/next';
+import { ZetkinUser } from '../../interfaces/ZetkinUser';
+
+const scaffoldOptions = {
+    localeScope: [
+        'layout.my',
+        'misc.publicHeader',
+        'pages.my',
+    ],
+};
+
+export const getServerSideProps : GetServerSideProps = scaffold(async () => {
+    return {
+        props: {},
```

Comment:
I think this should return a redirect to /my/feed. The component can be empty. Compare to what /login does.

Reply:
See my comment below.